src/test/repositories/in-memory-lesson-progress-repository.ts

```
@@ -249,23 +255,46 @@ export class InMemoryLessonProgressRepository implements ILessonProgressReposito
-    // For testing, we'll get lessons that belong to modules of this course
-    // This is a simplification - in real implementation this would come from DB joins
-    const courseLessons = this.lessons.filter(l => {
-      // Check if any stored lesson belongs to this course via its module
-      // In tests, we can track this via courseId if set, or fallback to empty
-      return l.courseId === courseId || (l as any).belongsToCourse === courseId;
-    });
-    const totalLessons = courseLessons.length;
+    // Get all modules for this course
+    let courseLessons: any[] = [];
+    let moduleIds: string[] = [];
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)